### DISCUSSION TITLE
Adding 'choices' within the option decorator

### DISCUSSION THREAD

**Discussion by BenSawary** [jmcdo29/nest-commander#326] [Q&A] [answered]
I'd like to define a valid list of a commands option argument. Since commander 7, (https://github.com/tj/commander.js/issues/518) it is possible to create an option with choices. Is it possible to add this functionality also in the nest-commander option decorator?

@Option({
    flags: '--option [option]',
    choices: ['a', 'b', 'c'],
    defaultValue: 'a'
  })
setOption(option: string) {}

**Comment by jmcdo29** [comment]
After briefly going through the linked issue, I think it definitely is possible to add this in. I'll try to work on it this weekend

**Comment by jmcdo29** [comment, marked as answer]
This is available in nest-commander @2.5.0
